content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -872,6 +872,50 @@ let layout = render_device.create_bind_group_layout(
 );
 ```
 
+## Unload rendering assets from RAM
+
+<div class="release-feature-authors">authors: @JMS55, @mockersf, @brianreavis</div>
+
+Meshes and the textures used to define their materials take up a ton of memory:
+in many games, memory usage is the biggest limitation on the resolution and polygon count of the game!
+Moreover, actually transferring that data from the CPU (and RAM) to the GPU (and VRAM) can be a real performance bottleneck,
+especially if you're doing it from scratch every frame.
+
+In the [initial PR] by @JMS55, Bevy added the ability to unload this data from the CPU, once it has been succesfully tranferred to the GPU.
+However, unloading the data from the CPU [can result in bugs], and make it harder to actually inspect or tweak that data from your other game systems.
+As a result, this behavior is currently off by default.
+
+To configure this behavior for your asset, set the [`RenderAssetUsages`] field when constructing your raw asset type such as `Image` or `Mesh`,
```

Comment:
Is there an easy way to do it for say a gltf model? Or were there problems with that?

Reply:
Looking at the changes in the linked PRs, it doesn't look like we expose a nice API for that currently :(